.github/workflows/build.yml

```diff
@@ -341,6 +351,45 @@ jobs:
         if: ${{ matrix.projects != 'dummy' }}
         run: ./scripts/run-in-parallel.sh docker-express
 
+  docker-static:
```

Comment:
we're up to 3 almost identical blocks (docker-static, docker-express and docker-next) might want to start thinking about extracting and re-using bits and pieces.

Reply:
Agreed and made the same comment in 3d6ac62b08c8db1e18a6a974ef13ad959ae36246. But the CI needs a bit of cleaning up anyway after the iterations it went through.